projects/uriparser/uri_dissect_query_malloc_fuzzer.cc

```diff
@@ -45,7 +45,9 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
   int chars_required;
   if (uriComposeQueryCharsRequiredA(query_list, &chars_required) != URI_SUCCESS)
     return 0;
-    
+
+  if (!chars_required) return 0;
```

Comment:
Could you initialize `chars_required` to 0 in line 45 above?  This way, we know it's initialized even if, for some reason, `uriComposeQueryCharsRequiredA` doesn't write to it.  Otherwise, this LGTM, thanks!